### ISSUE TITLE
When trying to migrate to AndroidX, the internal logger throws an error

### ISSUE THREAD

**Issue by gordonm867** [FIRST-Tech-Challenge/SkyStone#254]
The code uploads successfully, but the app crashes.

Logcat error text:

```
2020-07-29 17:45:00.624 18612-18641/com.qualcomm.ftcrobotcontroller E/AndroidRuntime: FATAL EXCEPTION: Logging Thread
    Process: com.qualcomm.ftcrobotcontroller, PID: 18612
    android.content.res.Resources$NotFoundException: String resource ID #0x7f0b0175
        at android.content.res.Resources.getText(Resources.java:365)
        at android.content.res.Resources.getString(Resources.java:411)
        at android.content.Context.getString(Context.java:476)
        at org.firstinspires.ftc.robotcore.internal.system.AppUtil.isRobotController(AppUtil.java:688)
        at com.qualcomm.robotcore.util.RobotLog.getLogFilename(RobotLog.java:662)
        at com.qualcomm.robotcore.util.RobotLog.getLogFile(RobotLog.java:719)
        at com.qualcomm.robotcore.util.RobotLog.access$000(RobotLog.java:57)
        at com.qualcomm.robotcore.util.RobotLog$1.run(RobotLog.java:555)
```

I was unable to diagnose anything using the line numbers given, and have no idea what the cause of this is.  Does anyone have any insights?

**Comment by FROGbots-4634** [comment]
Nothing immediately comes to mind... may I inquire as to why you're trying to use AndroidX?

**Comment by cmacfarl** [comment]
Fixed in v5.5
